brainstormforce/ultimate-addons-for-gutenberg
# Fix: Console warning on front end for slider library script

### Description
Fix Console warning on the front end for slider library javascript.

### Screenshot 
![cdaa259d-9b68-4e56-9792-60c7324a6291](https://user-images.githubusercontent.com/10000232/211769350-77bbe3a6-1828-47a2-b8bf-22724c43c1d5.png)


### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
- Add a slider to any page. 
- Update changes
- View the page on the front end while the console window is open. 
- There should not be any console warning as shown in the above screenshot

### Checklist:
